nodeup/pkg/model/protokube.go

```diff
@@ -251,6 +251,9 @@ func (t *ProtokubeBuilder) ProtokubeFlags(k8sVersion semver.Version) *ProtokubeF
 func (t *ProtokubeBuilder) ProtokubeEnvironmentVariables() string {
 	var buffer bytes.Buffer
 
+	// TODO write out an environments file for this.  This is getting a tad long.
```

Comment:
:+1: but no need to do as part of this PR.  We also are being a bit cavalier with escaping, so building up a map[string]string of env and then carefully writing it would help there also!

I think here we still need to pass `-e` flags though, as we're calling in to docker